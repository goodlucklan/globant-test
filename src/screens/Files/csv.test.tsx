import CSVScreen from "./Csv.screen";

describe("CSVScreen test", () => {
  test("render a Screen Component", () => {
    expect(<CSVScreen />).toMatchSnapshot();
  });
});
