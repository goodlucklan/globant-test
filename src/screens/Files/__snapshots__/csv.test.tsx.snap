// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSVScreen test render a Screen Component 1`] = `<CSVScreen />`;
